R/checkout.R

```diff
@@ -28,64 +55,142 @@ checkout <- function(repos = getOption("repos"),
   renv_project_lock(project = project)
 
   # set new repositories
+  repos <- repos %??% renv_checkout_repos(date)
   options(repos = repos)
 
+  # TODO: Activate Bioconductor if it appears to be used by this project
```

Comment:
Are there any other special considerations here for Bioconductor? (Could checking out an older snapshot also imply the need for an older Bioconductor release?) Feels like there could be some awkward conflicts.

Reply:
Yeah, I'm not sure how this should work for BioC given that it's mostly tied to a specific R version, not a point in time. Maybe a separate BioC argument that at lets you get them consistent with a given BioC version? It would default to the version corresponding to the current version of R?